support/user-guide/subaccounts.md

```diff
@@ -102,7 +104,9 @@ After creating a subaccount, the master account has the option of assigning a de
 
 **Permissions**
 
-Only master account users can create subaccounts. If you create a app user, they will have access to both the master account and subaccounts, so it is critical that be taken into account when provisioning app accounts on your system.
+Master account Admin and Developer users can create subaccounts. Master account Templates and Reporting users can see information about all subaccounts. Reporting Users can optionally be assigned to a specific subaccount instead of the master account - more information on this below in "Subaccount UI Access". It is important to consider if a user should have access to the master and all subaccounts or if a user should be restricted to a specific subaccount when provisioning app accounts on your system.
```

Comment:
imo you can get rid of the last sentence. Alternatively: "provisioning app accounts on your system" -> "inviting users".

Reply:
I didn't love that last sentence either, i waffled on taking it out or leaving it in. Agreed on taking it out.